roles/update/README.md

```diff
@@ -3,5 +3,6 @@ Role to run update
 
 ## Parameters
 * ``cifmw_update_extras`: (hash) Hold job variable that get set when running the update playbook.
-
-## Examples
+* ``cifmw_update_openstack_update_run_operators_updated`: (Boolean) Set if openstack_update_run make target should not modify openstack-operator csv to fake openstack services container change. Default to `True`.
+* ``cifmw_update_openstack_update_run_target_version`: (String) Define openstack target version to run update to.
```

Comment:
Hum ... the double "`" is strange and is introduced in L5, maybe we have to fix them ... in another review ?